qos-service-impl/build.gradle

```diff
@@ -51,14 +53,14 @@ configurations.all {
     resolutionStrategy {
         force 'org.hamcrest:hamcrest-core:1.3'
         force 'org.slf4j:slf4j-api:1.7.25'
+        force 'com.squareup.okhttp3:okhttp:3.8.1'
```

Comment:
We should include comments for why we're forcing any specific dependency version, and what the expectations are for when this force can be removed. This is especially important as downstream consumers may not force those same versions if we're not publishing an atlasdb BOM that consumers use to infer proper transitive dependency versions.